src/transport/MessageEndpoint.cpp

```diff
@@ -81,8 +81,8 @@ void MessageEndpoint::open(faabric::transport::MessageContext& context,
     }
 
     // Set socket options
-    this->socket->setsockopt(ZMQ_RCVTIMEO, recvTimeoutMs);
```

Comment:
This outputed a deprecated warning when I compiled it. The line in the source code where this is depcreated is [here](https://github.com/zeromq/cppzmq/blob/8e6066741349f4e9599d6178c432db9c82cfcdd2/zmq.hpp#L1655).